examples/ic_io/retimer_PI2EQX22024_output.json

```diff
@@ -0,0 +1,1786 @@
+{
+    "componentID": {
+        "partType": "retimer",
+        "manufacturer": "DIODES",
+        "componentName": "PI2EQX22024",
+        "orderableMPN": [
+            "PI2EQX22024XEAEX"
+        ],
+        "sourceDatasheetID": {
+            "publishedDate": "December, 2022",
```

Comment:
Based on discussion in WG today, we can change the dates in the example to match YYYY-MM-DD to match the specifications. 8-).

Reply:
done